internal/graphapi/models_test.go

```diff
@@ -127,3 +137,32 @@ func (g *GroupCleanup) MustDelete(ctx context.Context) {
 
 	EntClient.Group.DeleteOneID(g.GroupID).ExecX(ctx)
 }
+
+func (t *PersonalAccessTokenBuilder) MustNew(ctx context.Context) *generated.PersonalAccessToken {
+	ctx = privacy.DecisionContext(ctx, privacy.Allow)
+
+	if t.Name == "" {
+		t.Name = gofakeit.AppName()
+	}
+
+	if t.Description == "" {
+		t.Description = gofakeit.HipsterSentence(5)
+	}
+
+	owner := (&UserBuilder{}).MustNew(ctx)
+
+	if t.OwnerID == "" {
+		t.OwnerID = owner.ID
+	} else {
+		owner.ID = t.OwnerID
+	}
+
+	return EntClient.PersonalAccessToken.Create().
+		SetName(t.Name).
+		SetOwnerID(t.OwnerID).
+		SetOwner(owner).
+		SetToken(t.Token).
+		SetDescription(t.Description).
+		SetExpiresAt(t.ExpiresAt).
+		SaveX(ctx)
+}
```

Comment:
If t.OwnerID is set, we shouldn't need to create a new user as well? Do we need to set both `SetOwnerID` and `SetOwner` in the Create? 

```suggestion
	if t.OwnerID == "" {
		owner := (&UserBuilder{}).MustNew(ctx)
		t.OwnerID = owner.ID
	} 

	return EntClient.PersonalAccessToken.Create().
		SetName(t.Name).
		SetOwnerID(t.OwnerID).
		SetToken(t.Token).
		SetDescription(t.Description).
		SetExpiresAt(t.ExpiresAt).
		SaveX(ctx)
}
```